src/output.rs

```diff
@@ -130,8 +131,36 @@ impl fmt::Debug for ScaleWatcherHandle {
 }
 
 impl OutputState {
-    pub fn new() -> OutputState {
-        OutputState { xdg: GlobalProxy::new(), outputs: vec![], callbacks: vec![] }
+    pub fn new<
+        D: Dispatch<wl_output::WlOutput, OutputData>
+            + Dispatch<zxdg_output_v1::ZxdgOutputV1, OutputData>
+            + Dispatch<zxdg_output_manager_v1::ZxdgOutputManagerV1, GlobalData>
+            + 'static,
+    >(
+        global_list: &GlobalList,
+        qh: &QueueHandle<D>,
+    ) -> OutputState {
+        let (outputs, xdg) = global_list.contents().with_list(|globals| {
```

Comment:
Like seats, there is no notification via the handle trait that an outputs existed at the start. Same question but with outputs pretty much.

Reply:
The behavior here isn't really changed. `new_output()` isn't called until `done` is sent. So it still occurs basically as previously, I believe.